libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1043,7 +1043,29 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 	}
 	case FunctionType::Kind::Error:
```

Comment:
Should we just call `revertWithError` here?

Reply:
Interesting! I think I would rather simplify `revertWithError` for now.

Reply:
I just asked as it was looking almost the same like this block here. 
So simplifying sounds good to me

Reply:
I actually ended up calling `revertWithError` :)